apps/web/pages/v2/apps/installed/[category].tsx

```diff
@@ -123,6 +135,7 @@ const IntegrationsContainer = ({ variant, exclude }: IntegrationsContainerProps)
     calendar: Icon.FiCalendar,
     conferencing: Icon.FiVideo,
     payment: Icon.FiCreditCard,
+    automation: Icon.FiZap,
```

Comment:
This icon is same as Workflow. @PeerRich suggested me to use FiShare2 for automation.

Reply:
The Figma design uses the FiZap icon. @Jaibles What icon should we use for the Automation category in installed apps?

Reply:
ah right we agreed to use FiShare2 for automation

Reply:
Ok, I changed the icon. That's how it looks now:
<img width="950" alt="Screenshot 2022-10-14 at 15 33 26" src="https://user-images.githubusercontent.com/30310907/195859697-8587638b-2031-47c6-af41-039da99e167d.png">